parity/run.rs

```diff
@@ -133,6 +135,10 @@ pub fn execute(cmd: RunCmd) -> Result<(), String> {
 	// prepare account provider
 	let account_provider = Arc::new(try!(prepare_account_provider(&cmd.dirs, cmd.acc_conf)));
 
+	// create supervisor
+	let mut hypervisor = modules::hypervisor(Path::new(&cmd.dirs.ipc_path()));
+	modules::stratum(&mut hypervisor, &cmd.stratum);
```

Comment:
(so the case where the hypervisor or config is missing should be handled here, returning a proper string instead of panicking)